KCs-Linux-Scripts/Linux-Auto-Compile-Best-Of-Both-Worlds.sh

```diff
@@ -17,33 +18,33 @@ else
     echo "patch files already downloaded"
 fi
 
-if [ ! -d "/home/$(whoami)/ussedp/BestOfBothWorlds" ]
```

Comment:
as i said before whoami is also important because some distros like my own -f and -d will not work with ~

Reply:
did resolvyis